internal/oem/oem.go

```diff
@@ -103,10 +105,129 @@ func init() {
 		provider: gce.Creator{},
 		config: types.Config{
 			Systemd: types.Systemd{
-				Units: []types.SystemdUnit{{
-					Name:   "coreos-metadata-sshkeys@.service",
-					Enable: true,
-				}},
+				Units: []types.SystemdUnit{
+					{
+						Name:   "coreos-metadata-sshkeys@.service",
+						Enable: true,
+					},
+					{
+						Name:   "google-accounts-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
```

Comment:
Please install units into /usr/share/oem as part of the google daemon ebuild and so on instead and coping them, I don't want these to be scattered across so many repos.